Dockerfile

```diff
@@ -21,17 +22,13 @@ RUN chown -R bf /bio-formats-build
 USER bf
 WORKDIR /bio-formats-build
 RUN git submodule update --init
-
-USER bf
```

Comment:
Actually, one point regarding the dropping of the `bf` user for these commands is that the `m2` and any created files will have different `m2` locations and file ownership mismatching with the git clone and submodule update.  They all need to be the same user, for consistency and to avoid any breakage or duplication.

Reply:
As far as I can tell setting the USER and WORKDIR once should be sufficient for any commands that follow within the file